Cookbook/Technical-Documents/Development-Process.md

```diff
@@ -0,0 +1,61 @@
+# Development Process
+
+Babylon follows the [SSDLC](https://engineering.ops.babylontech.co.uk/docs/standards-ssdlc) when developing our software. To make the process easier, we use a few services to link code changes to tickets.
+
+There are 3 services we use daily:
+
+- Jira: this is where the development begins. Once a ticket has all the necessary details, dev can pick it up and implement it. <!-- TODO: [CNSMR-3230] link to Jira article -->
+- GitHub: used for hosting our project
+- CircleCI: our continuous integration service to run tests when integrating PRs <!-- TODO: [CNSMR-xxxx] link to CircleCI article -->
+
+## Jira 🎫
+
+- Everything we do must have a ticket
+- The first step is to assign the ticket to the engineer. Make sure it has all details and designs attached (if applicable)
+- The ticket is moved to `In progress` (this might be automatically done by the bot when branch published, if configured)
+
+## Get started with development 🛠
+
+- Create a branch where this work will live. Please follow this format: `[NAME]/[TICKET]-[DESCRIPTION (optional)]` (E.g. `john/TCKT-123-fix-bug`)
+- For new features, add a [local feature switch](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/FeatureSwitches.md) to avoid getting WIP code into the release. Add the name of this flag (and additional instructions) to the ticket to help QA test it
+- Implement all features, changes to satisfy the acceptance criteria defined in the ticket
```

Comment:
Would a link to some of our architecture patterns like Unidirectional flow, Reactive Swift, Bento, ReactiveFeedback etc under this section also be useful?  I think "Get Started with development"  could be split into "Git setup and feature flagging" and then "Development"(which includes links to best practices, architecture documentation, meeting A/C etc).

Reply:
Wasn't sure how detailed this article should be. I think if someone's looking for code guides, they can find it in the Table of Contents, otherwise when we create or edit articles, it'd need to be updated in both places (and there's a good chance it won't happen). I'd rather keep this simple and generic

Reply:
I agree with you Dan. We can mention:
> the implementation should be based in our current [Architecture](https://github.com/babylonhealth/babylon-ios/blob/develop/Documentation/README.md#architecture-)

Reply:
Just going to wait until #301 is merged so I can link to the section :)

Reply:
yeah makes sense :)